qa/rspec/cli/cli_tester.rb

```diff
@@ -0,0 +1,46 @@
+# encoding: utf-8
+require_relative "../../vagrant-helpers"
+require "logstash/devutils/rake" # provide download helper
+require "stud/temporary"
+
+module CliTester
+  DOWNLOAD_DIRECTORY = File.join(File.dirname(__FILE__), "..", "..", "vendor", "download")
+
+  def run_command(cmd, host)
+    hosts = (host.nil? ? servers : Array(host))
```

Comment:
I believe we are talking about the same thing, but our naming doesn't match.

If you look at this file https://github.com/elastic/logstash/blob/feature/acceptance_test/qa/rspec/commands/base.rb#L3,
This file define the `ServiceTester:Base` class but the path reference a `Command` module.

When I have answered your comment I looked at the content of the file not the path.